src/Checks/CacheHealthCheck.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace UKFast\HealthCheck\Checks;
+
+use Illuminate\Support\Facades\Cache;
+use UKFast\HealthCheck\HealthCheck;
+
+class CacheHealthCheck extends HealthCheck
+{
+    protected $name = 'cache';
+
+    protected $workingStores = [];
+    
+    protected $incorrectValues = [];
+
+    protected $exceptions = [];
+
+    public function status()
+    {
+        foreach (config('healthcheck.cache.stores') as $store) {
+            try {
+                $cache = Cache::store($store);
+
+                $cache->put('laravel-health-check', 'healthy', 1);
```

Comment:
I feel like there's a possibility it could expire before we pull it in weird circumstances. Is there another method that just connects?

Reply:
We could remove the 3rd param from the `put` method, which stores it indefinitely. 

The `pull` method retrieves and deletes the value, so this check would still clear up after itself.

Definitely open to suggestions here but I can't see another method which would just check it without actually doing a read/write.

Reply:
I think removing the third argument might be a good idea

Reply:
I'd swapped it to 60 mins but happy to make it last indefinitely? 

I couldn't decide which approach I preferred so I'll leave it to you.